devices/xiaomi.js

```diff
@@ -2409,6 +2573,37 @@ module.exports = [
             await endpoint.read('aqaraOpple', [0x040a], {manufacturerCode: 0x115f});
         },
     },
+    {
+        zigbeeModel: ['aqara.feeder.acn001'],
+        model: 'ZNCWWSQ01LM',
+        vendor: 'Xiaomi',
+        description: 'Aqara Pet Feeder C1',
+        fromZigbee: [fzLocal.aqara_feeder],
+        toZigbee: [tzLocal.aqara_feeder],
+        exposes: [
+            exposes.enum('feed', ea.STATE_SET, ['START']).withDescription('Start feeding'),
```

Comment:
Since this is an action we should not add it to the payload, so this should become `ea.SET` and `case 0x04150055:` can be removed.

Reply:
Not certainly in that way. the device sends a response with `0x04150055` when feeding - this is an indication that it has accepted the command. so I reset the current value so that I can set it again later

Reply:
Maybe we can change this to a switch then?

Reply:
I wanted to do exactly this as a button. if you change it to a switch, it will already look different ...

Reply:
do we now have any exposes that appears as an action button?